modules/unsupported/wfs-ng/src/test/resources/org/geotools/data/wfs/impl/test-data/Deegree_ri/1.1.0/GetCapabilities.xml

```diff
@@ -0,0 +1,277 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<WFS_Capabilities version="1.1.0" xmlns="http://www.opengis.net/wfs" xmlns:ows="http://www.opengis.net/ows" xmlns:ogc="http://www.opengis.net/ogc" xmlns:gml="http://www.opengis.net/gml" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.opengis.net/wfs http://schemas.opengis.net/wfs/1.1.0/wfs.xsd">
+  <ows:ServiceIdentification>
+    <ows:Title>deegree 3 CITE Demonstration Suite</ows:Title>
```

Comment:
When including test data from another organisation we need to:

a) Ask the other organisation if we are allowed to, they usually have a data license or something that spells out what we can and cannot do...
b) Make a note of where it came from in the notes (https://github.com/geotools/geotools/blob/master/modules/unsupported/wfs-ng/src/site/apt/review.apt) for the module

Often I have just constructed my own sample data rather than go through the hassle. For now can you provide the URL where you got this a sample data from, or we could contact Sebastian Goerke and ask for permission :)